featurebyte/routes/app_container_config.py

```diff
@@ -3,10 +3,30 @@
 
 This contains all our registrations for dependency injection.
 """
-from typing import List
+from typing import Dict, List
 
 from dataclasses import dataclass
 
+from featurebyte.enum import StrEnum
+
+
+class DepType(StrEnum):
+    """
+    DepType enums
+
+    They are prefixed with numbers to ensure that they are initialized in the correct order. For example, if a class
+    has dependencies on all 4 types, the dependencies will be initialized in the followingo rder
+    - no deps
+    - basic services
+    - services with extra deps
+    - controllers
+    """
+
+    NO_DEPS = "10_no_deps"
+    BASIC_SERVICE = "20_basic_service"
+    SERVICE_WITH_EXTRA_DEPS = "30_service_with_extra_deps"
+    CONTROLLER = "40_controller"
```

Comment:
We could use `featurebyte.enum.OrderedStrEnum` here.

Reply:
updated!